docs/client/orcid.rst

```diff
@@ -0,0 +1,44 @@
+.. _frameworks_clients:
+
+Authenticating with ORCID
+=================
+
+This documentation covers the special use case of authenticating with ORCID, a popular academic identity provider.
+
+ORCID has a non-standard response in the "amr" claim that returns a single string, rather than a list. Given the popularity of ORCID, this document specifies how to resolve this.
+
+Log In with ORCID
+---------------------
+Since Authlib 1.5.2 there is the functionality to pass a custom claims class to authorize_access_token.
+
+To do so, create a custom claims class:
```

Comment:
Doc syntax error, should be ending with `::`

Reply:
If you look at the "raw" version, it is: 

"To do so, create a custom claims class::" ?